### ISSUE TITLE
Test suite is failing and filling my inbox

### ISSUE THREAD

**Issue by jaraco** [cherrypy/cheroot#233]
The tests on GitHub Actions are failing and leaving lots of noise in my inbox. These alerts need to be suppressed or the tests need to be disabled until they're stable and less noisy.

**Comment by webknjaz** [comment]
Hey Jason, I suggest that you disable those notifications here for now: https://github.com/settings/notifications#notifications-github-actions-settings.

The long-term goal is to get those tests fixed, of course.

**Comment by jaraco** [comment]
Given that the Cheroot tests are flaky, having additional test suite runners (Appveyor, GitHub Actions, Circle CI, Travis, ...) magnifies any flakiness. Until we can get the false positives or intermittent failures down to less than 1/100, I think we need to pare down to the minumum number of test runners. Otherwise, every commit and every pull request will show a red X and the badges in the repo will always have something failing. Additionally, having all of these tests fills up the screen and adds delay to any contribution.

In other words, we need to refain from adding developer friction until the stability of the test suite is sufficient to support more services.

**Comment by webknjaz** [comment]
Yeah. The idea here is that all of these envs have some unique quirks because of being configured in a different manner. Some may have IPv6 configured properly/misconfigured/missing, for example. So I wanted to cover more cases.
I'd make some allowed to fail in order to reduce the friction.
